content/customize-dock.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Dock (Move or Customize)
+description: >
+  How to move the Dock to different areas of the screen and customize other settings.
+keywords:
+  - Launcher
+  - Bottom
+  - Left
+  - Moving
+  - Dock
+  - Dash to Dock
+  - Pop Dock
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+
+redirect_from:
+  - /movelauncher
+  - /articles/move-launcher
+---
+
+Pop!\_OS and Ubuntu both contain an application dock on the left side of the screen. You can adjust this preference, move the dock to the bottom of the screen, and more in <u>Settings</u>.
+
+## Default Dock on Pop!\_OS
+
+Pop!\_OS includes a lightweight version of Dash to Dock, which is why the dock already displays on the bottom of the screen by default.
+
+![Stock Pop Dock](/images/customize-dock/pop-dock.png)
+
+If you need more options or want to make more changes to the dock you can install Dash to Dock using the steps below.
+
+## Dash to Dock on Pop!\_OS
+
+When using Dash to Dock on Pop!\_OS it can effect the default Dock as it was forked from that project. To install Dash to Dock on Pop!\_OS, visit [the extension's page](https://extensions.gnome.org/extension/307/dash-to-dock/) in Firefox.
+
+![Dash to Dock page in Firefox](/images/customize-dock/dash-to-dock-page.png)
+
+If you see a purple message at the top about GNOME Shell integration, click the link to install the Firefox browser extension, then click the blue "Continue to Installation" button, and finally the blue "Add" button.
+
+![Installing the Firefox extension](/images/customize-dock/pop-install-dash-to-dock1.png)
+
+After refreshing the web page, you should see an On/Off switch at the top right of the page. Click the switch to toggle it to the "On" position.
+
+![Installing the Firefox extension](/images/customize-dock/pop-install-dash-to-dock2.png)
+
+On the resulting pop-up, click Install to download and install the extension.
+
+![Installing the Firefox extension](/images/customize-dock/pop-install-dash-to-dock3.png)
+
+At this point, Firefox can be closed. The dock will display on the bottom of the screen. You will want to disable COSMIC Dock in the <u>Extension</u> application as well.
+
+![Pop!_OS Extension application](/images/customize-dock/pop-gnome-extensions.png)
+
+To customize the dock's settings, click on the settings button for Dash to Dock in the <u>Extension</u> application"
+
+![Dash to Dock Settings](/images/customize-dock/pop-dash-to-dock-settings.png)
+
+In the Dash to Dock Settings window, you can choose which screen edge to display the launcher on, such as the bottom of the screen. You can also choose to extend the dock into "Panel mode," which looks similar to Ubuntu:
```

Comment:
>edge to display the launcher on

Probably want to change "launcher" to "dock" here. Launcher is still something else.